content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md

```diff
@@ -7,7 +7,7 @@ aliases:
  - /docs/tasks/telemetry/distributed-tracing/lightstep/
 ---
 
-This task shows you how to configure Istio to collect trace spans and send them to [LightStep Tracing](https://lightstep.com/products/) or [LightStep [𝑥]PM](https://lightstep.com/products/).
+This task shows you how to configure Istio to collect trace spans and send them to [LightStep Tracing](https://lightstep.com/product/) or [LightStep [𝑥]PM](https://lightstep.com/product/).
```

Comment:
It looks like the lightstep.com pages have been reorganized and now these, and other links in this doc, don't really go to appropriate parts of the site.

Maybe @robswhitmore can send a PR to properly fix this doc?

In the meantime, maybe we should just leave it as is. Now that #7014 has merged, it won't break the build anymore.

Reply:
Yes, our site recently changed. I'll get a PR going.